opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -2433,7 +2443,9 @@ namespace Opm
         // TODO: we should decide whether to keep the updated well_state, or recover to use the old well_state
         if (converged) {
             std::ostringstream sstr;
-            sstr << " well " << name() << " manage to get converged within " << it << " inner iterations";
+            sstr << " well " << name() << " manage to get converged within " << it << " inner iterations \n";
+            if (relax_convergence)
+                sstr << "A relaxed tolerance is used after 40 iterations";
```

Comment:
This should be "after [StrictInnerIterMsWells] iterations" now I guess?

Reply:
I will update it asap